tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -112,5 +119,24 @@ sub test_flags() {
     return {fatal => 1};
 }
 
+sub post_fail_hook() {
+    my $self = shift;
+
+    if (get_var("DESKTOP") =~ /textmode|minimalx/) {
+        send_key "ctrl-alt-f1";
```

Comment:
you define a log-console in lib/susedistribution. tty3 and 5 are still free to use for such purposes.